mindsdb_sdk/connectors/rest_api.py

```diff
@@ -88,13 +86,11 @@ def projects(self):
         return pd.DataFrame(r.json())
 
     @_try_relogin
-    def model_predict(self, project, model, data, params=None, version=None):
+    def model_predict(self, project, model, data, params={}, version=None):
```

Comment:
Dict in default value can be mutable. I we modify it its new value will be kept in param default.
Now we don't modify it but we can add some validations in this code in the future and forget that it is mutable.
Lets keep the old code (None in default)

Reply:
Ok, got it! Thanks for pointing that out